docs/modules/ROOT/pages/mgmt_services/node_metadata_manager.adoc

```diff
@@ -1,19 +1,40 @@
 = Node Metadata Manager Design Specification
-Eunju Kim <eunju.kim@futurewei.com>, Liguang Xie <lxie@futurewei.com>
+Eunju Kim <eunju.kim@futurewei.com>, Liguang Xie <lxie@futurewei.com>, Prasad Kommoju <pkommoju@futurewei.com>
 v0.1, 2020-04-30
+v.02, 2021-03-15
 :toc: right
 
-NOTE: This document is under development
+NOTE: This revision includes only the new API.
 
 == Overview
 
-The node manager is a control plane component responsible of managing and configurating physical servers/nodes/hosts inside a data center.
+The node manager is a control plane component responsible for managing and configuring physical servers/nodes/hosts inside a data center. This is referred to as NMM in this and other documents.
```

Comment:
Besides physical hosts, we are also starting managing registration of infrastructure components like NCM.